test/replace_using_package_version_test.py

```diff
@@ -94,9 +94,9 @@ def test_find_match_in_version(self):
     def test_find_package_version(self, mock_run):
         mock_run.return_value = b'2.3.1'
         assert find_package_version('package', '/foo') == '2.3.1'
-        mock_run.called_once_with([
+        mock_run.assert_called_once_with([
```

Comment:
ouch... 🤦🏽  I guess I am the one to blame here. I think I saw similar issues in KIWI. The created mock was not complaining on non existing method and it silently when unnoticed.

Thanks for the fix :+1: